slog/README.MD

```diff
@@ -46,24 +46,26 @@ func main() {
 
 	ctx := context.Background()
 	handler := sentryslog.Option{
-		LogLevel:   slog.LevelInfo,
-		EventLevel: slog.LevelError,
+		EventLevel: []slog.Level{slog.LevelError, sentryslog.LevelFatal}, // Only Error and Fatal as events
+		LogLevel:   []slog.Level{slog.LevelWarn, slog.LevelInfo},         // Only Warn and Info as logs
 	}.NewSentryHandler(ctx)
     logger := slog.New(handler)
 
 	// Example logging
-	logger.Info("This will be sent to sentry as a log")
-	logger.Error("An error occurred", "user", "test-user") // this will be sent both as an event & log
+	logger.Info("This will be sent to sentry as a Log entry")
+	logger.Error("An error occurred", "user", "test-user") // this will be sent as an Event
+	// These will be ignored
+	logger.Debug("This will be ignored")
 }
 ```
 
 ## Configuration
 
 The slog-sentry package offers several options to customize how logs are handled and sent to Sentry. These are specified through the Option struct:
 
-- `EventLevel`: Minimum level to send logs as events to Sentry. Defaults to `slog.LevelError`.
+- `EventLevel`: Slice of specific levels to send `Events` to Sentry. Defaults to `[]slog.Level{slog.LevelError, LevelFatal}`.
 
-- `LogLevel`: Minimum level to send logs to Sentry. Defaults to `slog.LevelDebug`.
+- `LogLevel`: Slice of specific levels to send `Log` entries to Sentry. Defaults to `[]slog.Level{slog.LevelError, LevelFatal}`.
```

Comment:
Is this right? From below I can see `o.LogLevel = []slog.Level{slog.LevelDebug, slog.LevelInfo, slog.LevelWarn, slog.LevelError, LevelFatal}`